packages/web-console/src/utils/isCloud.ts

```diff
@@ -0,0 +1,3 @@
+export const isCloud = () => {
+  return /questdb\.(net|com)$/.test(window.location.hostname)
```

Comment:
this is a sub-optimal way to find out if copy is supported, i will port 

```sql
show server_conf
```

to OSS and web console could call that instead

Reply:
Very good, just ping me when this is available and I'll refactor the code 👍

Reply:
Implemented as part of `show parameters`.

Reply:
Closest I could get to an appropriate fix looks something like:

```js
import { useContext, useEffect, useState } from 'react'
import { QuestContext } from "../providers"

type Result = {
  'property_path': string
  'env_var_name': string
  'value': string | null
  'value_source': string
  'sensitive': string
  'dynamic': 'false'
}

export const isCopyEnabled = () => {
  const { quest } = useContext(QuestContext)
  const [value, setValue] = useState(false)
  useEffect(() => {
    void quest.query<Result>(`(show parameters) where property_path ilike 'cairo.sql.copy.root'`)
    .then((result => {
      if (result.type === 'dql' && result.count > 0) {
        console.log(result.data)
        setValue(result.data[0].value !== null)
      }
    }))
  }, [quest])

  return value
}
```

No idea if this will actually be helpful for you @insmac, but I'd like to get more knowledgeable on how to actually work with the parts of the web console that interact with the backend.

Reply:
or, like, idk,

```sql
with
data as (show parameters)
select value = null as is_disabled from data
where
  property_path ilike 'cairo.sql.copy.root'
```